shared/src/db/database.spec.ts

```diff
@@ -830,4 +836,40 @@ describe("baseDatabase.ts", () => {
             });
         });
     });
+    // it("deletes expired documents when not in cms-mode", async () => {
```

Comment:
This commented out test is still here. I think it is important that we have a test for the delete expired functionality.

Perhaps one way would be to `new` up a `database`, but first override the `config.cms` value? Not sure if calling `initLuminaryShared` again in here would work, but maybe that's also a way to go.

Reply:
Implemented